cli/src/util/native-run.ts

```diff
@@ -23,7 +23,7 @@ export async function getPlatformTargets(platformName: string): Promise<Platform
   try {
     const output = await runNativeRun([platformName, '--list', '--json']);
     const parsedOutput = JSON.parse(output);
-    if (parsedOutput.devices.length || parsedOutput.virtualDevices.length) {
+    if (parsedOutput.devices.length >= 0 || parsedOutput.virtualDevices.length >= 0) {
```

Comment:
length can be 0 if native-run returned some error, with this change native-run errors won't be shown anymore as it will always be `>= 0`.

native-run doesn't throw errors, they are returned in `parsedOutput.errors` object, so better check if its length is 0 in the else and log the error there.

Reply:
So instead of the native error we'd get `logger.info(`No target devices found for ${platformName}.\n`);` instead?

Reply:
this is for when there are no native-run errors, which happens when there are no devices, when there is an error it would show the native-run error

Reply:
Yeah, that's the best we can do for now - probably have to take a look at native-run one of these days. Knowing apple - we're gonna be forced to next month.